docs/testnet/Introduction.md

```diff
@@ -1,16 +1,12 @@
-# Introducing Obscuro Testnet
-The Obscuro Testnet is a public network used by developers, node operators and end users to test Obscuro before deploying to Mainnet.
+# Introducing Oikos
```

Comment:
I'm with Gavin. Somewhere here, you have to say that Oikos is a testnet.

Reply:
Can we choose a better name that is easier to pronounce? We have the entire universe of language at our disposal so lets make its something easy (Oi-kos? o-i-kos? oik-os?)

Reply:
To make the pronunciation crystal clear, let's spell it `οἶκος`.

Reply:
I don't really like Oikos TBH. So I'm with you guys, let's choose a better name

Reply:
Evan's Cat